packages/react/src/Popover/Popover.module.css

```diff
@@ -205,4 +205,71 @@
       bottom: calc(var(--base-size-16) + 1px);
     }
   }
+
+  &:where([data-width='xsmall']) {
+    width: 192px;
```

Comment:
I think we should be using CSS variables, not hard-coded numbers. So `var(--overlay-width-xsmall)` instead of `192px`. The rest of the variables can be found on the Primitives page: https://primer.style/product/primitives/size/#overlay